packages/filter/src/features/search-filter/stories/SearchFilter.stories.tsx

```diff
@@ -391,6 +392,194 @@ export const Demo = () => {
+          spacing={2}
+          indent
+          shadow={"popover"}
+          zIndex={1}
+        >
+          <PrimaryButton label={"Search"} />
+        </Row>
+      </SearchFilterDrawer>
+    </SearchFilterContext>
+  );
+};
+
+export const WithClearFiltersInHeader = () => {
+  const { dispatch, actions, state } = useLocalSearchFilterState<
+    SalesItemSearchFilterModel,
+    SalesItemSearchFilterSectionKey
+  >(createSearchFilterInitialState({ divisions: {}, categories: {} }));
+
+  return (
+    <SearchFilterContext state={state} actions={actions} dispatch={dispatch}>
+      <Card>
+        <Row
+          alignItems={"center"}
+          justifyContent={"space-between"}
+          indent={2}
+          spacing
+          minHeight={"56px"}
+        >
+          <Row alignItems={"center"}>
+            <Heading style={{ whiteSpace: "nowrap" }}>App name</Heading>
+            <Indent />
+            <SearchFilterButton />
+            <Indent num={0.5} />
+            <SearchFilterChips>
+              <SectionChips
+                sectionId={"comparisonDate"}
+                emptyChipLabel={"No dates"}
+                {...createChipsPropsForDateRange(
+                  state.formModel,
+                  "startDate",
+                  "endDate"
+                )}
+              />
+              <SectionChips
+                sectionId={"divisions"}
+                emptyChipLabel={"All divisions"}
+                {...createChipsPropsForBooleanRecord(
+                  state.formModel,
+                  "divisions",
+                  divisionOptions
+                )}
+              />
+              <SectionChips
+                sectionId={"categories"}
+                emptyChipLabel={"All categories"}
+                {...createChipsPropsForBooleanRecord(
+                  state.formModel,
+                  "categories",
+                  categoryOptions
+                )}
+              />
+            </SearchFilterChips>
+          </Row>
+        </Row>
+      </Card>
+      <SearchFilterDrawer headerContentRight={<ClearFiltersButton />}>
+        <DateRangeCalendarSection
+          sectionId={"comparisonDate"}
+          {...createDateRangeSectionProps(
+            state.formModel,
+            "startDate",
+            "endDate"
+          )}
+        />
+        <ChipMultiSelectSection
+          sectionId={"divisions"}
+          loading={false}
+          noneSelectedLabel={"All divisions"}
+          options={divisionOptions}
+          value={state.formModel.divisions}
+          onValueChange={(divisions) =>
+            dispatch(actions.setFormModelFields({ divisions }))
+          }
+        />
+        <SimpleCheckboxListSection
+          sectionId={"categories"}
+          options={categoryOptions}
+          value={state.formModel.categories}
+          onValueChange={(categories) =>
+            dispatch(actions.setFormModelFields({ categories }))
+          }
```

Comment:
I would like to add a tag example as well. Something like this:

```suggestion
          }
          contentRight={
            selectedCategories > 0 && (
              <Tag label={selectedCategories.toString()} />
            )
          }
```

And define `selectedCategories`:
```
  const selectedCategories = Object.values(state.formModel.categories).filter(
    (value) => value
  ).length;
```

Maybe remove the other filters just to avoid confusion ("the other ones don't work!"), or if you want to you could add tags for them as well to show what they would look like (might actually be a better idea).